documentation/source/library_reference.rst

```diff
@@ -269,3 +269,14 @@ XGMII
     :members:
     :member-order: bysource
     :show-inheritance:
+
+Miscellaneous
+=============
+
+WaveDrom
```

Comment:
Please call this already "A signal tracer for WaveDrom". The section title otherwise sounds like we provide wavedrom here, which we don't.

Reply:
Yes, it is intended, with this the following code will be shown as code properly.

Reply:
Argh, replied to the wrong comment. I will change the heading of course.

Reply:
Oh, I had forgotten about that... I'll be without access to a computer until the end of the next week unfortunately.